container/extractConfig.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017 The Bazel Authors. All rights reserved.
```

Comment:
Could you please add this logic [here](https://github.com/bazelbuild/rules_docker/blob/master/container/go/cmd/extract_config/main.go) instead?